frontend/js/src/utils/APIService.ts

```diff
@@ -1822,4 +1822,21 @@ export default class APIService {
     await this.checkStatus(response);
     return response.json();
   };
+
+  getPlaylistCoverArt = async (
```

Comment:
This is now unused, with the server endpoint removed.

I think the endpoint could actually be useful. In particular, I'm thinking about the grid view on the playlists page, where we don't have all the playlist metadata but we could show each playlist's cover art.

Also thinking about atom feeds where we sometimes need a URL to the SVG rather than SVG content.

Reply:
So, LMK do a thing. I'll remove the frontend function to call the server api, and bring back the server endpoint.

Reply:
Currently I am getting an error about image size when I try the endpoint on test:
"image size must be between 128 and 1024, inclusive."

URL was POST https://test-api.listenbrainz.org/1/art/playlist/5bb057eb-b38a-4a8d-8218-c5004f052fb2/250/1/

On a side note: why did you want a POST endpoint rather than GET?
My envisioned used would be a simple `<img src="https://api.listenbrainz.org/1/art/playlist/${playlist.identifier}/250/1/">` or something along those lines in the PlaylistCard component to get the image directly, but that would require a GET endpoint.

What was your use case?

Reply:
So the endpoint's structure is `/playlist/<uuid:playlist_mbid>/<int:dimension>/<int:layout>`. The dimension and layout are based on those defined in the `GRID_TILE_DESIGNS`. 

Since, a lot of playlists are private, we cannot remove authentication from the request, so therefore we cannot add it as a src in the image HTML Tag.

Reply:
Ah bummer, I forgot about that. Yeah, I guess we fetch the SVG content with a fetch call then :+1:

Still, the image_size issue was not related to the dimension or layout params, not sure what is happening there. Did you successfully test a POST request with auth on the test-api.lb art endpoint?

Reply:
Yes. POST https://test-api.listenbrainz.org/1/art/playlist/5bb057eb-b38a-4a8d-8218-c5004f052fb2/4/0

Reply:
It is indeed working. :tada: 
Not sure what I did wrong before ¯\_(ツ)_/¯